.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+name: build
+
+on:
+  push:
+    branches: ["main"]
+  release:
+    types: ["published"]
+  pull_request:
+    branches: ["main"]
+
+env:
+  GOLANG_VERSION: "1.21"
+
+jobs:
+  build:
+    name: Build ${{ matrix.variant.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        variant:
+          - {"name": "cosign_i386", "args": "GOARCH=386"}
+          - {"name": "cosign_amd64", "args": "GOARCH=amd64"}
+          - {"name": "cosign_armhf", "args": "GOARM=6 GOARCH=arm"}
+          - {"name": "cosign_armv7", "args": "GOARM=7 GOARCH=arm"}
+          - {"name": "cosign_aarch64", "args": "GOARCH=arm64"}
+    steps:
+    - name: Checkout the repository
```

Comment:
It's a nit but I prefer to have list items indented as well. I see we generally stick to it in other repos too.

Reply:
Run it through "Format Document" using the "Prettier" formatter, which added the indentation for list items as well.